apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorItemInfo.vue

```diff
@@ -44,10 +44,12 @@
                                 </span>
                             </p>
                         </div>
-                        <!-- TODO: add in-progress state -->
-                        <span v-else-if="props.item">
+                        <span v-else-if="props.item.recentJobAnalyze[props.item.recentJobAnalyze.length - 1].status === JOB_STATE.IN_PROGRESS">
```

Comment:
issue: I think there are cases where the `props.item` is `undefined`. How about writing it as computed?